.cirrus.yml

```diff
@@ -75,8 +75,12 @@ doccheck_task:
       "${SKOPEO_PATH}/${SCRIPT_BASE}/runner.sh" doccheck
 
 osx_task:
-    # Run for regular PRs and those with [CI:BUILD] but not [CI:DOCS]
-    only_if: &not_docs_multiarch >-
+    # Don't run for docs-only or multi-arch image builds.
+    # Also don't run on release-branches or their PRs,
+    # since base container-image is not version-constrained.
+    only_if: &not_docs_or_release_branch >-
```

Comment:
There are now two definitions of `not_docs_or_release_branch`. Is that necessary?

Reply:
Ugh, nope not good, it's a bug.  I'll fix.